logstash-core-event-java/src/test/java/com/logstash/KeyNodeTest.java

```diff
@@ -0,0 +1,27 @@
+package com.logstash;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class KeyNodeTest {
```

Comment:
I think it makes sense. I will verify to see if we have test cases for list null values and add one if its missing and run it on both the ruby and java implementations and make sure the java impl is consistent.